webdriver/tests/user_prompts/send_alert_text.py

```diff
@@ -0,0 +1,67 @@
+from tests.support.asserts import assert_error, assert_success
+
+
+def send_alert_text(session, body=None):
+    return session.transport.send("POST", "session/{session_id}/alert/text"
+                                  .format(session_id=session.session_id), body)
+
+# Used to check if alert text sent was correct
+def accept_alert(session):
+    return session.transport.send("POST", "session/{session_id}/alert/accept"
+                                  .format(session_id=session.session_id))
```

Comment:
You don’t have to make a raw call to this endpoint. We only do that for the command under test, which is Send Alert Text.

Reply:
fixed